.travis.yml

```diff
@@ -34,6 +35,7 @@ jobs:
       os: windows
```

Comment:
Note: I'm working on adding support for extra python modules on FreeBSD and Windows on Jenkins. I'll let you know when that's done.